packages/react-app-polyfill/ie9.js

```diff
@@ -11,4 +11,7 @@ require('./ie11');
 // React 16+ relies on Map, Set, and requestAnimationFrame
 require('core-js/features/map');
```

Comment:
Similarly we should probably change https://github.com/facebook/create-react-app/blob/56d1de274576f8e5371d83f07a87c32473c4ae9a/packages/react-app-polyfill/ie11.js#L14 to reference `self` instead of `window`, which allows it to work in a WebWorker context.